### ISSUE TITLE
flutter doc has an unexpected `>`

### ISSUE THREAD

**Issue by HannibalKcc** [flutter/flutter#67802]
[doc -> Navigator class -> pages property](https://master-api.flutter.dev/flutter/widgets/Navigator/pages.html)

has an unexpected `>` in Implementation
> final List> pages

but i review [source code](https://github.com/flutter/flutter/blob/da92fc92e9/packages/flutter/lib/src/widgets/navigator.dart#L1481) ,it's right
> final List<Page<dynamic>> pages;

**Comment by Abhishek01039** [comment]
Hi, @darshankawar can you please provide the source code of this?
so I will change it and make PR.
Thanks.

**Comment by Sangwan100** [comment]
hello @darshankawar  iam beginner here and i want to contribute in  this , can you please guide me in doing this.
thanks.

**Comment by bassel** [comment]
Here's how the html looks like:

```html
<code class="language-dart hljs">
    <span class="hljs-keyword">final</span> 
    <span class="hljs-built_in">List</span>
    <page<dynamic>&gt; pages</page<dynamic>
</code>
```

It happens in other places as well, here are some examples:
-  [localizationsDelegates property](https://master-api.flutter.dev/flutter/widgets/WidgetsApp/localizationsDelegates.html)
-  [delegates property](https://master-api.flutter.dev/flutter/widgets/Localizations/delegates.html)
-  [Cupertino App actions property](https://master-api.flutter.dev/flutter/cupertino/CupertinoApp/actions.html)

So basically it's happening with the properties that have a type of the form `A<B<C>>` where `<C>` is being rendered as an HTML tag.
